relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkg.py

```diff
@@ -350,20 +350,21 @@ def upgrade_all(mode):
         currentVersion = rpkg.PluginVersion(utils.DB["plugins"][p]["version"])
         pkgs = plugin.Plugin(p)
         pkgs.getAvailablePackages()
+        latestversion = None
```

Comment:
I would have put currentVersion in latestVersion, so we don't have to check if it is None